### ISSUE TITLE
Problem after Insteon Express server is started

### ISSUE THREAD

**Issue by efthymakis** [kuestess/homebridge-platform-insteonlocal#245]
I see the following in the Hoobs log file, after which all Insteon devices are no longer accessible in Home App.

5/11/2022, 3:57:26 PM [InsteonLocal] Insteon event listener started...
5/11/2022, 3:57:26 PM [InsteonLocal] Started Insteon Express server...
5/11/2022, 3:57:26 PM [UI] Hub/PLM info is {
5/11/2022, 3:57:26 PM raw: '026046F7980333A5',
5/11/2022, 3:57:26 PM type: '60',
5/11/2022, 3:57:26 PM id: '46F798',
5/11/2022, 3:57:26 PM deviceCategory: { id: 3, name: 'Network Bridges' },
5/11/2022, 3:57:26 PM deviceSubcategory: { id: 51 },
5/11/2022, 3:57:26 PM firmwareVersion: 'A5'
5/11/2022, 3:57:26 PM }

Nothing changed in the environment.  No updates or firmware changes.
I rebooted Hoobs and the Insteon hub with no change.

Any ideas?

**Comment by kuestess** [comment]
@efthymakis That all looks normal.  Did you clear the cache in Hoobs or remove the bridge from the Home app?  Does the bridge still show in the Home app?

**Comment by efthymakis** [comment]
I did not clear the cache or remove the Hoobs hub from the Home App.  I just reconfirmed to be sure.

This may be a Hoobs issue as the last thing I see in the log is what I posted.  No events show up later in the log.  Same pattern after I reboot/restart Hoobs or services.

**Comment by kuestess** [comment]
Can you post your config?

**Comment by efthymakis** [comment]
Here is the complete config, including the Insteon plugin:

{
    "server": {
        "port": 8080,
        "autostart": 30,
        "home_setup_id": "X-HM://0023ISYWY9SDM",
        "polling_seconds": 2,
        "origin": "*"
    },
    "client": {
        "default_route": "status",
        "inactive_logoff": 30,
        "theme": "hoobs-dark",
        "locale": "en",
        "temp_units": "fahrenheit",
        "country_code": "US",
        "postal_code": "11358"
    },
    "bridge": {
        "name": "HOOBS",
        "username": "CC:22:3D:E3:CE:30",
        "port": 51826,
        "pin": "031-45-154"
    },
    "description": "This file is used for plugin configuration.",
    "ports": {},
    "accessories": [],
    "platforms": [
        {
            "platform": "Somfy myLink",
            "systemID": "mySystem",
            "ipAddress": "192.168.1.20",
            "reverseAll": true
        },
        {
            "platform": "InsteonLocal",
            "user": "Krea7707",
            "pass": "Macedonia",
            "host": "192.168.1.162",
            "port": "25105",
            "model": "2245",
            "server_port": "3000",
            "devices": [
                {
                    "name": "Deck",
                    "deviceID": "4C29C9",
                    "dimmable": "yes",
                    "deviceType": "dimmer"
                },
                {
                    "name": "Patio",
                    "deviceID": "42AEAA",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Rear Landscape",
                    "deviceID": "424B6E",
                    "deviceType": "switch",
                    "dimmable": "no"
                },
                {
                    "name": "Basement Back",
                    "deviceID": "413474",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Basement Front",
                    "deviceID": "43A295",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Side Entrance Primary",
                    "deviceID": "42AE98",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Bathroom Light",
                    "deviceID": "412D62",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Fan",
                    "deviceID": "4352AB",
                    "deviceType": "switch",
                    "dimmable": "no"
                },
                {
                    "name": "Hallway Primary",
                    "deviceID": "42B02F",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Foyer",
                    "deviceID": "42AF67",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Front Landscape",
                    "deviceID": "424834",
                    "deviceType": "switch",
                    "dimmable": "no"
                },
                {
                    "name": "Holiday",
                    "deviceID": "42513C",
                    "deviceType": "switch",
                    "dimmable": "no"
                },
                {
                    "name": "Porch",
                    "deviceID": "42C41A",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Garage Light",
                    "deviceID": "4CC43B",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Kitchen Primary",
                    "deviceID": "42C72C",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Kitchen Table",
                    "deviceID": "42BE9A",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Under Cabinet",
                    "deviceID": "42C855",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Living Room",
                    "deviceID": "42BBB5",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Top Outlet",
                    "deviceID": "440D7E",
                    "deviceType": "outlet",
                    "position": "top",
                    "dimmable": "no"
                },
                {
                    "name": "Bottom Outlet",
                    "deviceID": "440D7E",
                    "deviceType": "outlet",
                    "position": "bottom",
                    "dimmable": "no"
                },
                {
                    "name": "Dining Room",
                    "deviceID": "3E2412",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Entrance Primary",
                    "deviceID": "42B467",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Sitting Room",
                    "deviceID": "42AEB8",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Hall Primary",
                    "deviceID": "42BB45",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Hallway Secondary",
                    "deviceID": "42BEEA",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Entrance Secondary",
                    "deviceID": "42AEC7",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Kitchen Secondary",
                    "deviceID": "42BBD1",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Side Entrance Secondary",
                    "deviceID": "41347C",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                },
                {
                    "name": "Hall Secondary",
                    "deviceID": "42AF04",
                    "deviceType": "dimmer",
                    "dimmable": "yes"
                }
            ],
            "plugin_map": {
                "plugin_name": "homebridge-platform-insteonlocal"
            },
            "name": "InsteonLocal",
            "refresh": "0",
            "keepAlive": "3600"
        },
        {
            "platform": "Ring",
            "plugin_map": {
                "plugin_name": "homebridge-ring"
            },
            "refreshToken": "eyJhbGciOiJIUzUxMiIsImprdSI6Ii9vYXV0aC9pbnRlcm5hbC9qd2tzIiwia2lkIjoiYzEyODEwMGIiLCJ0eXAiOiJKV1QifQ.eyJpYXQiOjE2NTIxMzQ1MDcsImlzcyI6IlJpbmdPYXV0aFNlcnZpY2UtcHJvZDp1cy1lYXN0LTE6M2M3ZjhlNGIiLCJyZWZyZXNoX2NpZCI6InJpbmdfb2ZmaWNpYWxfYW5kcm9pZCIsInJlZnJlc2hfc2NvcGVzIjpbImNsaWVudCJdLCJyZWZyZXNoX3VzZXJfaWQiOjMwNjQzMzksInJuZCI6Ik5xbjR6M1N1V3d2VEp3Iiwic2Vzc2lvbl9pZCI6IjhlMWQzM2M5LTBjMWItNDEwZi05YzQxLWUyNTRiZDUyMDk5NCIsInR5cGUiOiJyZWZyZXNoLXRva2VuIn0.BZeAxKW3P9V0e4IJ5P8mvV3vpfWGc1eEvYLqP5pp4pIjZ6u38nfdjdhfZvRmIvI7Xq5jhxRCJLTswExlLw5Dxg",
            "hideLightGroups": true,
            "hideDoorbellSwitch": false,
            "hideCameraMotionSensor": false,
            "hideCameraSirenSwitch": true,
            "hideInHomeDoorbellSwitch": true,
            "hideAlarmSirenSwitch": true,
            "hideUnsupportedServices": true,
            "avoidSnapshotBatteryDrain": false,
            "showPanicButtons": false,
            "sendCameraMotionNotificationsToTv": true,
            "sendDoorbellMotionNotificationsToTv": false,
            "debug": false,
            "cameraStatusPollingSeconds": 10,
            "beamDurationSeconds": 32767
        },
        {
            "platform": "Rachio-Platform",
            "plugin_map": {
                "plugin_name": "homebridge-rachio-platform"
            },
            "name": "Rachio",
            "clear_previous_webhooks": false,
            "api_key": "3e4c9e99-f2c2-4391-8854-13ee2742e51f",
            "internal_webhook_port": 27546,
            "external_webhook_address": "http://74.101.21.5:12453"
        }
    ]
}

**Comment by efthymakis** [comment]
I opened an issue with Hoobs.  The service won't stay up, explaining why I don't see anything in the logs after the last thing I posted.
Looking through the logs, I see the following just after initialization:


5/11/2022, 5:47:01 PM Error loading platform requested in your config.json at position 3
5/11/2022, 5:47:01 PM Got SIGTERM, shutting down Bridge...

No changes to configs or environment so not sure why it's acting up now.
